sendgrid/helpers/permissions/permissions.py

```diff
@@ -0,0 +1,305 @@
+            return True
+        else:
+            print("Error: Invalid group name")
+            return False
+
+
+    #   Updates permissions for a permissions list
+    #   inputs:
+    #       list_name:string variable that contains the name of list
+    #   outputs:
+    #       success:boolean True if operation is successful, False if not
+    def update_permissions_list(self, list_name, permissions):
+        if (list_name in self.available_api_key_permissions):
+            self.api_key_permissions[list_name] = permissions
+            return True
+        else:
+            print("Error: Invalid list name")
+            return False
+
+    
+    #   Updates permissions for a permissions group
+    #   inputs:
+    #       group_name:string variable that contains the name of a group
+    #   outputs:
+    #       success:boolean True if operation is sucessful, False if not
+    def update_permissions_group(self, group_name, permissions):
+        if (group_name in self.available_common_api_key_groups):
+            self.common_api_key_groups[group_name] = permissions
+            return True
+        else:
+            print("Error: Invalid group name")
+            return False
+
+
+    #   Reads permissions that a permissions list contains
+    #   inputs:
+    #       list_name:string variable that contains the name of a list
+    #   outputs:
+    #       permissions:string[] variable that contains the permissions of a list,
+    #       or None if no list name found
+    def read_permissions_from_list(self, list_name):
+        if (list_name in self.available_api_key_permissions):
+            return (self.api_key_permissions[list_name])
+        else:
+            print(
+                "There is no permissions list with that given name {0}".format(list_name))
+            return None
+
+
+    #   Read permissions that a permissions group contains
+    #   inputs:
+    #       group_name:string variable that contains the name of a group
+    #   outputs:
+    #       permissions:string[] variable that contains the permissions of a group,
+    #       or None if no group name found
+    def read_permissions_from_group(self, group_name):
+        if (group_name in self.available_common_api_key_groups):
+            return (self.common_api_key_groups[group_name])
+        else:
+            print(
+                "There is no permissions group with that given name {0}".format(group_name))
+            return None
+
+    '''
+        Requested features in second bullet point:
+        Allow for creating an array of all the scopes possible, as a "create admin permissions" method
+    '''
+
+
+    #   Create an admin list - a list containing all available permissions
+    #   outputs:
+    #       admin_list:string[] variable containing all possible permissions
+    def create_admin_permission_list(self):
+        if (self.admin_api_list):
+            return self.admin_api_list
+        else:
+            self.admin_api_list = []
```

Comment:
Instead of using a list, you could use a [`set`](https://docs.python.org/3/library/stdtypes.html#set-types-set-frozenset), a handy data structure that's basically a `dict` without values.  Then you could say
```python
self.admin_api_permissions = set()
for group in self.available_api_key_permissions:
    self.admin_api_permissions.update(group)
```